mailbox/api/src/main/java/org/apache/james/mailbox/model/CurrentQuotas.java

```diff
@@ -24,6 +24,8 @@
 import org.apache.james.core.quota.QuotaCountUsage;
```

Comment:
Is this commit still a WIP?

Reply:
Nope, I changed it but didn't push it yet